1. Don't use -D flag to install which is a GNU extension not available
   with BSD install. https://github.com/rwos/gti/pull/58

2. Allow prefix to be set via variable. Upstream has taken a different
   approach after 1.7.0 which is wrong and will need to be corrected /
   reverted.
--- Makefile.orig	2020-02-09 07:59:44.000000000 -0600
+++ Makefile	2022-03-30 20:49:47.000000000 -0500
@@ -3,11 +3,12 @@
 CC=cc
 CFLAGS+=-O2 -std=c89 -Wpedantic -Wall -Wextra -Wunused -Wshadow -Wdouble-promotion -Wstrict-overflow=5
 
-INSTALL=install -D
+INSTALL=install
 INSTALL_DATA=$(INSTALL) -m 644
+PREFIX=/usr
 
-BINDIR=$(DESTDIR)/usr/bin
-MANDIR=$(DESTDIR)/usr/share/man/man6
+BINDIR=$(DESTDIR)$(PREFIX)/bin
+MANDIR=$(DESTDIR)$(PREFIX)/share/man/man6
 
 STRIP=strip
 ifeq ($(OS),Windows_NT)
